subgraphs/maker/src/maker.ts

```diff
@@ -1,10 +1,9 @@
-import { Address, ByteArray, log } from '@graphprotocol/graph-ts'
-import { BIG_DECIMAL_1E18, BIG_DECIMAL_ZERO, FACTORY_ADDRESS, SUSHI_BAR_ADDRESS, SUSHI_MAKER_ADDRESS } from 'const'
+import { Address, log } from '@graphprotocol/graph-ts'
+import { BIG_DECIMAL_ZERO, FACTORY_ADDRESS, SUSHI_MAKER_ADDRESS } from '../../../packages/constants'
 import { Maker, Server, Serving } from '../generated/schema'
-
-import { ERC20 as ERC20Contract } from '../generated/Maker/ERC20'
 import { Factory as FactoryContract } from '../generated/Maker/Factory'
-import { Swap as SwapEvent } from '../generated/Maker/Pair'
+import { LogConvert as LogConvertEvent } from '../generated/Maker/Maker'
+
 
 function getMaker(): Maker {
```

Comment:
Let's move these get() calls into an `src/entities` directory similar to how minichef and some of the other subgraphs are setup.